driver-core/src/main/com/mongodb/client/model/CreateCollectionOptions.java

```diff
@@ -194,6 +199,72 @@ public CreateCollectionOptions collation(@Nullable final Collation collation) {
         return this;
     }
 
+    /**
+     * Returns the expire-after option.  The default value is 0, which indicates no expiration.
+     *
+     * @param timeUnit the time unit
+     * @return the expire-after option, which may be null.
+     * @since 4.3
+     * @mongodb.driver.manual core/timeseries-collections/ Time-series collections
+     */
+    @Nullable
+    public long getExpireAfter(final TimeUnit timeUnit) {
```

Comment:
The method is not `@Nullable` anymore.

P.S. My previous comment was placed terribly, moved here.

Reply:
Whoops!  Done.

(I feel like that should generate a warning of some sort, to mark a method returning a primitive value as Nullable)

Reply:
IDEA marks the line with a warning, but SpotBugs indeed could have enforced the rule, but it does not have one: https://spotbugs.readthedocs.io/en/latest/bugDescriptions.html